### DISCUSSION TITLE
Why there is a different outcome for same operation?

### DISCUSSION THREAD

**Discussion by dikshanshs** [neperfepx/neper#257] [Q&A] [answered]
**_@rquey sir, 
for the below two commands the output should be the same but they are different (Image attached)_**

**$neper -V ../simul.sim \
    -simstep 10 \
    -datanodecoo coo \
    -dataelt1drad 0.004 \
    -dataelt3dedgerad 0.0015 \
    -datanodecoofact 10 \
    -dataelt3dcol ori \
    -dataelt3dcolscheme "ipf(z)" \
    -dataeltscaletitle "Work pl" \
    -showelt1d all \
    -imagesize 800:800 \
    -print 10_ori**
Resulting Image for above input
![image](https://user-images.githubusercontent.com/85054451/123678632-2f7d4480-d864-11eb-8c76-cff15efcfc68.png)

**$neper -V simul.sim -loop STEP 0 1 10 -simstep STEP -datanodecoo coo \
-dataeltcol ori -dataeltcolscheme "ipf(z)" -print imgSTEP -endloop**

Resulting Image for above input
![image](https://user-images.githubusercontent.com/85054451/123678589-1b394780-d864-11eb-9123-d064deb2542b.png)

**Comment by rquey** [comment, marked as answer]
You used `-datanodecoofact 10` in the first one and not the second...
